Core/Requests/RegisterRequest.cs

```diff
@@ -13,6 +13,8 @@ public class RegisterRequest
         public string Email { get; set; }
 
         [Required(ErrorMessage = "Password is required")]
+        [RegularExpression(@"^(?=.*[0-9])(?=.*[!@#$%^&*])(?=.*[A-Z])[0-9a-zA-Z!@#$%^&*0-9]{7,}$",
+            ErrorMessage = "Password is not valid (at least one special sign, digit, upper letter and lenght >= 7)")]
```

Comment:
You've added this regex in appsettings, read it from those. Also, you might want to add the validationErrorMessage in appsettings too. Or use the Identity password configuration.